ci/build-linux.sh

```diff
@@ -59,8 +61,8 @@ sudo docker run \
      --env TERM="${TERM:-dumb}" \
      --user "${docker_uid}" \
      --volume "${PWD}":/v \
-     --volume "${PWD}/build-output/cache":/.cache \
-     --volume "${PWD}/build-output/ccache":/.ccache \
+     --volume "${PWD}/build-output/cache":${docker_home}/.cache \
+     --volume "${PWD}/build-output/ccache":${docker_home}/.ccache \
```

Comment:
What happens when `${docker_home}` == `""`, and this is referring to `/.cache` — should that be `$HOME/.ccache` instead?

Reply:
Inside the docker image we are running with the uid of the user calling the program, but not creating an actual user inside the docker image for that uid.  Therefore there is no `$HOME` for that poor uid.